src/sidepanel/v2/components/ChatInput.tsx

```
@@ -152,6 +133,17 @@ export function ChatInput({ isConnected, isProcessing }: ChatInputProps) {
     setShowTabSelector(false)
     textareaRef.current?.focus()
   }
+
+  const handleTabSelected = (tabId: number) => {
```

ESLint

[error] 'tabId' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)